src/resolvers/address.resolver.ts

```diff
@@ -49,3 +49,21 @@ export class RecipientAddressResolver implements Resolver {
         return AccountService.getRecipient(resolution);
     }
 }
+
+export class TargetAddressResolver implements Resolver {
```

Comment:
Now, we should be able to reuse the ``AddressResolver``, dropping the ``TargetAddressResolver`` class. See this example:

command modifications: https://github.com/nemtech/nem2-cli/commit/259f84990ce172ebb052f4727b9819f669448775#diff-e6859f147b2f078f73d5ac96f14f8ac4R55-R57

resolver modifications: https://github.com/nemtech/nem2-cli/commit/259f84990ce172ebb052f4727b9819f669448775#diff-08865a78c570237b3d5887ffdb253c07R21-R27